src/rt/cover.d

```diff
@@ -149,10 +149,80 @@ uint parseNum(const(char)[] s)
 T min(T)(T a, T b) { return a < b ? a : b; }
 T max(T)(T a, T b) { return b < a ? a : b; }
 
+version (unittest) __gshared bool inUnittest;
+
+string parseMergeOpt(string opt) @nogc nothrow
+{
+    switch (opt)
+    {
+        case "no", "No", "NO", "n", "N", "0":
+            merge = false;
+            return "0";
+        case "", "yes", "Yes", "YES", "y", "Y", "1":
+            merge = true;
+            return "1";
+        default:
+            if (!inUnittest)
```

Comment:
Given this here, this would not compile outside unittests, right? Why not `version (unittest)` on this method?

Reply:
What do you mean? Why wouldn't it compile outside unittests?

Reply:
Am I not seeing right, or the variable `inUnittest` is only defined if `version (unittest)` (see few lines above)?

Reply:
I'm not the only one, travis is also complaining here :-)

Reply:
Right, I see it now. Copy&paste error :)

Reply:
Updated.